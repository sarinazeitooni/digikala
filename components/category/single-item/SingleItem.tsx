import React from 'react';
import SwiperCore, {Autoplay} from 'swiper/core';
import { Swiper, SwiperSlide } from "swiper/react";
import data from "../../specials/texts/data";
import CategoryCard from "../category-card/CategoryCard";
const SingleItem = () => {
    SwiperCore.use([Autoplay]);
    return (
        <Swiper spaceBetween={30} centeredSlides={true} autoplay={{
            "delay": 3500,
            "disableOnInteraction": false
        }} className="mySwiper">
            {data.cards.map((item)=>{
                return(
                    <SwiperSlide>
                        <CategoryCard hasDiscount = {item.hasDiscount} discount={item.discount} img={item.img} price={item.price} title={item.title}/>
                    </SwiperSlide>
                )
            })}
        </Swiper>
    )
}
export default SingleItem;