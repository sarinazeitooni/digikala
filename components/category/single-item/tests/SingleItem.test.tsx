import {shallow} from "enzyme";
import SingleItem from "../SingleItem";
describe("SingleItem component" , ()=>{
    it("should render correctly" , ()=>{
        const container = shallow(<SingleItem/>);
        expect(container).toMatchSnapshot();
    })
});