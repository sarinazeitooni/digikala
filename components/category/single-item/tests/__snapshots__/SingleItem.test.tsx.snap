// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SingleItem component should render correctly 1`] = `ShallowWrapper {}`;
